internal/build/native-tests/src/test/java/org/jdbi/v3/nativetests/StubTest.java

```diff
@@ -0,0 +1,25 @@
+/*
```

Comment:
the path is wrong IMHO. "build" is a maven module, not a folder to add a module to. This should be "internal/native-tests" or even a toplevel module such as e2e.